core/src/main/java/hudson/model/queue/CauseOfBlockage.java

```diff
@@ -104,6 +108,33 @@ public void print(TaskListener listener) {
     }
 
     /**
+     * Build is blocked because a node (or its retention strategy) is not accepting tasks.
+     * @since FIXME
+     */
+    public static final class BecauseNodeIsNotAcceptingTasks extends CauseOfBlockage implements NeedsMoreExecutor {
+
+        public final Node node;
```

Comment:
`CauseOfBlockage`s are transient. This class is directly analogous to others in the same class like `BecauseNodeIsOffline`, which have long retained a `Node` field.